app/Http/Controllers/HomeController.php

```diff
@@ -1,9 +1,10 @@
 <?php
-
 namespace App\Http\Controllers;
-
 use Illuminate\Http\Request;
-
+use App\User;
+use App\Blog;
```

Comment:
Please remove this, this is not needed here.

Reply:
It has been deleted.